### PR TITLE
New Drazil plushie (inverse lizard plushie)

### PR DESCRIPTION
**Pull request by ArtisticRoomba**
## About the PR
Adds new plushie named "drazil plushie", juice that makes you Hew reagent, and Hew sound.

It can be found in maints lockers, albeit with a very low chance.

This PR also adds localization support to the Weh and Hew emotes.

## Why / Balance
image

I'm going to be honest, this is a shitpost I got too carried away with. But, if it passes review... ¯\_(ツ)_/¯
## Technical details
Just YAML changes.

## Media
Content Client_OvAyHBB9Q8
Content Client_HpwFzVHZPs
Content Client_uz3wkYbg0h
Content Client_dHstn1wUQK

https://github.com/user-attachments/assets/319e4ad1-c1e6-47fa-a3a4-80a5fec905a8

## Requirements

- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.

**Changelog**
:cl:
- add: New plushie, named "drazil plushie". Based on an inverted lizard plushie. It can be rarely found in maints lockers. Hew!
- add: New reagent, Juice that makes you Hew. It can be made from juicing inverted lizard plushies. Hew!

### CODE REVIEW

**Comment by ps3moira** [comment]
Should be for April fools otherwise bloat content blegh

**Comment by ArtisticRoomba** [comment]
> Should be for April fools otherwise bloat content blegh

I agree. Up to maintainers.

**Comment by notquitehadouken** [comment]
Evil weh be like

**Comment by SlimmSlamm** [comment]
> Should be for April fools otherwise bloat content blegh

Content is content.

**Comment by Djungelskog2** [comment]
give the juice that makes you hew have a really dangerous or atleast unique reaction with juice that makes you weh (possible future PR godo)

**Comment by ArtisticRoomba** [comment]
> give the juice that makes you hew have a really dangerous or atleast unique reaction with juice that makes you weh (possible future PR godo)

I was considering creating an inverted lizard it would polymorph you into if you hit the cap but... I got lazy. Might consider putting it in later.

**Review by UbaserB — APPROVED** [review]
sprite approval, I love how it looks

**Comment by Hrosts** [comment]
CL typo:

> drazid plushie

File: Resources/Prototypes/Voice/speech_emotes.yml [hunk 1]
```diff
@@ -336,6 +336,13 @@
   icon: Interface/Emotes/weh.png
   chatMessages: [Wehs!]
 
+- type: emote
+  id: Hew
+  name: chat-emote-name-hew
+  category: Vocal
+  icon: Interface/Emotes/hew.png
+  chatMessages: [Hews!]
```
Comment (lzk228) [inline Resources/Prototypes/Voice/speech_emotes.yml:344]: since you are here, could you make locale support to it and to weh above it

Comment (ArtisticRoomba) [reply Resources/Prototypes/Voice/speech_emotes.yml]: Added localization support, tested, works ingame.

File: Resources/Locale/en-US/chat/emotes.ftl [hunk 2]
```diff
@@ -50,6 +51,8 @@ chat-emote-msg-gasp = gasps.
 chat-emote-msg-deathgasp = seizes up and falls limp, {POSS-ADJ($entity)} eyes dead and lifeless...
 chat-emote-msg-deathgasp-monkey = lets out a faint chimper as {SUBJECT($entity)} collapses and stops moving...
 chat-emote-msg-buzz = buzzes!
+chat-emote-msg-weh = Wehs!
+chat-emote-msg-hew = Hews!
```
[the remark this answers is not in the document]
Comment (ArtisticRoomba) [reply Resources/Locale/en-US/chat/emotes.ftl:55]: Isn't it funny that the Wehs! emote is the only one that is capitalized though? It's been like that forever. I guess it was just a mistake, but do we still want to change it?

[the remark this answers is not in the document]
Comment (ArtisticRoomba) [reply Resources/Locale/en-US/chat/emotes.ftl:55]: after looking at gitblame I can conclude it was done in error, I am changing it now

**Comment by Everturning** [comment]
make the chemical be a "liztless" chemical instead, like an antonym to "vibrant"

**Comment by ScarKy0** [comment]
I can't believe someone actually did this

**Comment by ArtisticRoomba** [comment]
> give the juice that makes you hew have a really dangerous or atleast unique reaction with juice that makes you weh (possible future PR godo)

Could make it explode. It is like antimatter and matter colliding after all. Would be balanced too considering it's a really rare plushie.

**Pull request merged**